src/components/safe-apps/SafeAppsSDKLink/index.tsx

```diff
@@ -1,49 +1,43 @@
-import { useState } from 'react'
-import Link from 'next/link'
-import Box from '@mui/material/Box'
-import Collapse from '@mui/material/Collapse'
-import Typography from '@mui/material/Typography'
-import MUILink from '@mui/material/Link'
-import Fab from '@mui/material/Fab'
+import { useEffect, useState } from 'react'
+import { Fab, Typography } from '@mui/material'
 import KeyboardDoubleArrowUpRoundedIcon from '@mui/icons-material/KeyboardDoubleArrowUpRounded'
-
+import classnames from 'classnames'
 import CodeIcon from '@/public/images/apps/code-icon.svg'
 import { SAFE_APPS_SDK_DOCS_URL } from '@/config/constants'
 import css from './styles.module.css'
+import ExternalLink from '@/components/common/ExternalLink'
 
 const SafeAppsSDKLink = () => {
-  const [openModal, setOpenModal] = useState<boolean>(false)
+  const [mini, setMini] = useState(false)
+
+  useEffect(() => {
+    const minScroll = 130
```

Comment:
Nit: can we align this with constant styles?

```suggestion
    const MIN_SCROLL = 130
```